corehq/sql_accessors/sql_templates/save_case_and_related_models_2.sql

```diff
@@ -44,6 +44,8 @@ BEGIN
             closed_on = commcarecase.closed_on,
             closed_by = commcarecase.closed_by,
             deleted = commcarecase.deleted,
+            deleted_on = commcarecase.deleted_on,
+            deletion_id = commcarecase.deletion_id,
```

Comment:
were there no tests for this? (i don't actually know that we ever use these but still surprised)